spec/controllers/bibliographic_controller_spec.rb

```diff
@@ -308,6 +308,24 @@
       end
     end
 
+    context "sortable call number " do
+      it "handles normal call numbers" do
+        stub_alma_bib_items(mms_id: "9965126093506421", filename: "cdl_9965126093506421_bib_items.json")
+        get :bib_items, params: { bib_id: "9965126093506421" }, format: 'json'
+        expect(response.status).to be 200
+        body = JSON.parse(response.body)
+        expect(body["firestone$stacks"].first["sortable_call_number"]).to eq "PS.3558.A62424.B43--2010"
```

Comment:
Where is `--` coming from? It seems like it should just be a space.

Reply:
That's what the normalized coded does, I would be surprised if the normalized code uses spaces since it would make it harder for humans to compare values (computers wouldn't care, but humans could get confused with spaces)

Reply:
okay, cool!